modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcRequestHandler.java

```diff
@@ -506,6 +507,10 @@ private JdbcResponse processBulkLoadFileBatch(JdbcBulkLoadBatchRequest req) {
         // Write node id.
         if (protocolVer.compareTo(VER_2_8_0) >= 0)
             writer.writeUuid(connCtx.kernalContext().localNodeId());
+
+        // Write all features supported by the node.
+        if (protocolVer.compareTo(VER_2_8_2) >= 0)
+            writer.writeByteArray(JdbcThinFeature.allFeatures());
```

Comment:
Matching should happen in a single place, not on both client and server. Or we can face de-synchronisation as we update our clients and servers.

Reply:
Fixed.